src/app/app.module.ts

```diff
@@ -120,11 +122,13 @@ import { AppAuthGuard } from './guards/auth.guard';
         source: 'assets'
       }
     },
+    ...AppGuardFactory.guards,
```

Comment:
Is that needed? I think it will always be empty array or will have dependency on the order of calls